manhunt-base/src/main/java/io/github/ytg1234/manhunt/mixin/MinecraftServerMixin.java

```diff
@@ -0,0 +1,82 @@
+package io.github.ytg1234.manhunt.mixin;
+
+import com.mojang.authlib.GameProfileRepository;
+import com.mojang.authlib.minecraft.MinecraftSessionService;
+import com.mojang.datafixers.DataFixer;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.resource.ResourcePackManager;
+import net.minecraft.resource.ServerResourceManager;
+import net.minecraft.server.MinecraftServer;
+
+import net.minecraft.server.WorldGenerationProgressListenerFactory;
+import net.minecraft.util.UserCache;
+import net.minecraft.util.registry.DynamicRegistryManager;
+import net.minecraft.world.SaveProperties;
+import net.minecraft.world.level.storage.LevelStorage;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.net.Proxy;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Mixin(MinecraftServer.class)
+class MinecraftServerMixin implements io.github.ytg1234.manhunt.util.ManhuntServer {
+    @Unique @NotNull  private List<UUID> manhuntBase_hunters;
+    @Unique @Nullable private UUID manhuntBase_speedRunner;
+    @Unique @NotNull  private List<PlayerEntity> manhuntBase_haveMod;
+
+    @Deprecated MinecraftServerMixin() {
+        manhuntBase_haveMod = new ArrayList<>();
+        manhuntBase_hunters = new ArrayList<>();    // Stupid IDEA
+    }
+
+    @Inject(method = "<init>", at = @At("RETURN"))
+    private void OnConstructor(
```

Comment:
I usually like to describe what the method does, for example `initializeFields`. Also, we use camelCase.